FlashLFQ/Bayesian Protein Quantification/ProteinQuantificationEngine.cs

```diff
@@ -568,7 +568,7 @@ private double DetermineExperimentalNull(string treatmentCondition)
                 else
                 {
                     // TODO: not sure what to do here.. there were no protein quant results for this condition
-                    // throw an exception... or just make the cutoff 1.0?
+                    // for now the cutoff is just made to be a fold-change of 1.0, but maybe an exception should be thrown...
                     stdDev = 0.3333;
```

Comment:
Where does this value come from?

Reply:
it's arbitrary. if there are no proteins to determine a fold-change, 1.0 is used to continue with the algorithm without crashing. the user can choose to re-run with a manually input cutoff if they want

Reply:
My math's a little rusty, but 0.3333 != 1.0

Reply:
the fold-change cutoff is 3 * stdDev

Reply:
Can you add that info in a comment here?